modules/grpc/1.70.0/MODULE.bazel

```diff
@@ -0,0 +1,91 @@
+# Copyright 2025 the gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+module(
+    name = "grpc",
+    version = "1.70.0",
+    compatibility_level = 1,
+    repo_name = "com_github_grpc_grpc",
+)
+
+bazel_dep(name = "abseil-cpp", version = "20240722.0", repo_name = "com_google_absl")
+bazel_dep(name = "apple_support", version = "1.17.1", repo_name = "build_bazel_apple_support")
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "boringssl", version = "0.20241024.0")  # mistmatched 20241211
```

Comment:
is the plan to update these separately in grpc's own MODULE.bazel file first?

Reply:
If you're asking about the "mismatched" comment, it refers to different dependency versions between bzlmod and the workspace.  This is due to unregistered versions in BCR and requires to be addressed in the future.